src/app/Fake.DotNet.Cli/CreateProcessExt.fs

```diff
@@ -0,0 +1,121 @@
+namespace Fake.Core
```

Comment:
Should we mode this to Fake.DotNet (might solve the AutoOpen issue?

Reply:
As above, for consistency with the other `CreateProcess` extension, so they all look like they're part of the base module.